genienlp/data_utils/embeddings.py

```diff
@@ -214,7 +213,7 @@ def load_embeddings(cachedir, context_emb_names, question_emb_names, decoder_emb
             continue
 
         emb_type = get_embedding_type(emb_name)
-        if _is_bert(emb_name) or _is_xlmr(emb_name):
+        if emb_name in ALL_PRETRAINED_MODEL_ARCHIVE_MAP.keys():
```

Comment:
Don't you need Tokenizer subclasses for new models?

Reply:
So this is meant to be used with "cache-embeddings" command to cache models locally or for the docker. The model and tokenizer will be instantiated from "run_lm_finetune" and "run_generation" script.

Reply:
Yeah then we don't need it, because the docker doesn't cache the pretrained models anymore.

Reply:
Mbart models are pretty big and it's better to cache them (in our own dockers). It speeds up subsequent training.

Reply:
You still have to download the docker image on each machine for each training. The difference is not that big...

Reply:
Agreed